src/doc/man/cargo-package.md

```diff
@@ -32,6 +32,16 @@ stored in the `target/package` directory. This performs the following steps:
     - A `.cargo_vcs_info.json` file is included that contains information
       about the current VCS checkout hash if available, as well as a flag if the
       worktree is dirty.
+    - Symlinks are flattened to their target files.
+    - Subdirectories containing `Cargo.toml` are excluded, as they are assumed
+      to be separate packages.
+    - A `target` directory in the root of the package is excluded.
+    - Files and directories matching the `[include]` and `[exclude]` fields in
+      the manifest are included or excluded. Include patterns can override the
+      following rules.
+    - Files that are ignored by VCS are excluded.
+    - Unix hidden files and directories (starting with `.`) are excluded.
```

Comment:
Thank you. I love your writing!

Yet, I just recalled that we already have the rule written down in <https://doc.rust-lang.org/nightly/cargo/reference/manifest.html#the-exclude-and-include-fields>.

Instead of duplicating the writing, what about providing a relative link to `../reference/manifest.html#the-exclude-and-include-fields`? Something like:

```suggestion
    - Files and directories are included or excluded based on rules mentioned in
      [the `[include]` and `[exclude]` fields](../reference/manifest.html#the-exclude-and-include-fields).
```